openedx/core/lib/django_courseware_routers.py

```diff
@@ -51,7 +51,7 @@ def allow_migrate(self, db, app_label, model_name=None, **hints):  # pylint: dis
         """
         if model_name is not None:
             model = hints.get('model')
-            if self._is_csmh(model):
+            if model is not None and self._is_csmh(model):
```

Comment:
My temporary fix is [logically identical](https://github.com/open-craft/edx-platform/commit/faa2b2f34e14f85874b32f07ef353a083d297149), so :+1: !